src/lib.rs

```diff
@@ -117,13 +117,24 @@ pub fn run_loop() {
 /// Starts an app mounted to a body of the document.
```

Comment:
Maybe you should mention `start_app_with_props` in the comment of the `start_app` function?